index.html

```diff
@@ -0,0 +1,627 @@
+                        <div class="card-heading">
+                            <img src="./images/images/program_icon_04.png" alt="" />
+                            <h3>cc exhibition</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/images/program_icon_03.png" alt="" />
+                            <h3>forum</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/images/program_icon_04.png" alt="" />
+                            <h3>Workshop</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/images/program_icon_05.png" alt="" />
+                            <h3>cc party</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-1"></div>
+                    <div class="col-10 join_summit">
+                        <center>Join summit 2023</center>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row down_part">
+            <div class="col-12">
+                <center>
+                    <span>
+                        <a href="##">See whole program </a>
+                    </span>
+                </center>
+            </div>
+        </div>
+    </section>
+
+    <!--Down section-->
+    <section class="container down_section" id="desktop_speakers">
```

Comment:
- [ ] These speaker cards' text overlaps their images from `769px` to `1200px`. kindly consider fixing them.
- [ ] Also consider making sure that the name, position, and description, all follow the font-sizes and font-weights of the original design. right now they looks different from the original design.
your design
![t](https://user-images.githubusercontent.com/96683843/226440868-d8439416-9c53-4972-b6d8-6d9325510587.jpg)

original design
![j](https://user-images.githubusercontent.com/96683843/226440966-a561c1bb-c70e-4374-ba9d-7f41d698c406.jpg)